config/collectdata/python/write_organ_tags.py

```diff
@@ -2,13 +2,28 @@
 # generate_organ_tags.py v1.0.0
 
 # ACTION:
-#  Generate list of IRIs and labels from UBERON organ_slim and output in JSON format
+#  Generate list of IRIs and labels from UBERON organ_slim and output in JSON format to YAML file
 #  Output will be used to filter and/or boost search results in Cell Annotation Platform (CAP)
 
+# INSTRUCTIONS:
+#  Organ IRIs and labels will be directed to a file named neo4j2owl-config.yaml by default
+#  To direct output to a differnt file, execute by running:
+#   python write_organ_tags.py -f file_name
+#   Replace 'file_name' with name of YAML file
```

Comment:
By default, python's argparse allows for `--help` as a flag. This is useful here as well.